client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -108,7 +119,16 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 					}
 				/>
 			</Column>
-			<Column mobileHidden>{ site.plan.product_name_short }</Column>
+			<Column mobileHidden>
+				<SitePlan>
+					{ site.jetpack && ! isAtomicSite && (
```

Comment:
Do we need to include `isAtomicSite` in the conditional? Aren't all Atomic sites using Jetpack too?

Reply:
They are using Jetpack, but they are not using Jetpack plans. Without that check, we would display the Jetpack icon with "Business", "Premium," and other standard WPCOM plans.

Reply:
Right! That makes sense.